cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/SubstationConversion.java

```diff
@@ -36,7 +36,7 @@ public void convert() {
 
         Country country = CountryConversion.fromRegionName(regionName)
                 .orElseGet(() -> CountryConversion.fromSubregionName(subRegionName)
-                        .orElseGet(() -> CountryConversion.defaultCountry(this)));
+                        .orElse(null));
```

Comment:
Is it better to work with a null country or to work with an optional<country>. It depends on what it’s done with this country. Could you check that? Maybe it’s better to get the country from the optional later

Reply:
It is only used to set the country of the substation. By ensuring than if it is not present, it is null, we ensure than the set country is null.